lib/discover/global.go

```diff
@@ -46,8 +45,7 @@ const (
 )
 
 type announcement struct {
-	Direct []string `json:"direct"`
-	Relays []Relay  `json:"relays"`
+	Addresses []string `json:"addresses"`
```

Comment:
Yeah this is fine. Just now that we live on HTTPS, I don't want to just bump us to port 444 instead. But we can handle the details server side.